ui/packages/consul-ui/app/components/topology-metrics/index.js

```diff
@@ -84,6 +84,12 @@ export default class TopologyMetrics extends Component {
     return items;
   }
 
+  get mainNotIngressService() {
+    const kind = get(this.args.service.Service, 'Kind') || '';
```

Comment:
Is `Kind` ever falsey here?

Reply:
no, Kind doesn't return falsey. It simply doesn't return `Kind` making this line undefined. Should I remove the ```|| ''``` ?